main/index.js

```diff
@@ -21,6 +22,13 @@ const { ipcMain } = require('electron/main')
 const inTest = (process.env.NODE_ENV === 'test')
 const isDev = !app.isPackaged && !inTest
 
+// Expose additional metadata for Electron preload script
```

Comment:
Is overwriting `process.env` the best way to pass data to the client? 🤔

Reply:
Tbh I haven't thought about how to do this in a setup where the renderer doesn't have access to `require('electron')`

Reply:
The preload script is intentionally sandboxed for security reasons. It has access to a very limited set of APIs, see https://www.electronjs.org/docs/latest/tutorial/tutorial-preload

I am open to use a different mechanism for passing data from `main` to `preload`, what do you suggest?

Reply:
I don't have anything on the top of my mind, I was more curious if there were well known alternatives. Thanks for the context!